library/core/src/slice/mod.rs

```diff
@@ -6660,6 +6587,79 @@ unsafe impl<'a, T> TrustedRandomAccess for RChunksExactMut<'a, T> {
 // Free functions
 //
 
+/// Converts a range over this slice to [`Range`].
```

Comment:
Since it's a free function now there's no "this slice" anymore.

Reply:
Ah, my edit there got lost during rebasing... changed to
```
/// Performs bounds-checking of the given range.
/// The returned [`Range`] is safe to pass to [`get_unchecked`] and [`get_unchecked_mut`]
/// for slices of the given length.
```